keyboards/ymdk_bface/config.h

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2019 Ethan Durrant (emdarcher)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define VENDOR_ID       0x20A0
+#define PRODUCT_ID      0x422D
+// You can edit those at usbconfig.h about line 250. These values will
+// unforunatly be ignored so far
+#define MANUFACTURER    YMDK
+#define PRODUCT         bface
+
+/* matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+//                        0   1   2   3   4   5   6   7   8   9   A   B   C   D   E
+#define MATRIX_ROW_PINS { B7, B6, B5, B4, B3}
+#define MATRIX_COL_PINS { A0, A1, A2, A3, A4, A5, A6, A7, C7, C6, C5, C4, C3, C2, D7}
+#define UNUSED_PINS
+
+#define NO_UART 1
+
+#ifdef BACKLIGHT_ENABLE
+	// the backlight PWM does not work (yet). Therefore, we only have two backlight levels (on/off)
```

Comment:
The Backlight feature should work now, D4 is supported on the 32A: https://docs.qmk.fm/#/feature_backlight?id=caveats

Reply:
Ok, I will try to change this. I do not have backlight LEDs on my PCB to test with right now, but can add more backlight level options to the code. This was copied from the B.face config.h, so that board might need updating as well to use backlight levels.